### PR TITLE
Update doco to reflect SourceForge to GitHub code migration

### PR DESCRIPTION
**Pull request by apjanke**
The GeSHi code repo has been migrated over from SourceForge to GitHub, right?

This PR updates the "Downloading GeSHi" section with instructions for cloning the repo from GitHub instead of SourceForge/SVN.

### CODE REVIEW

File: docs/geshi-doc.html [hunk 1]
```diff
@@ -616,7 +616,7 @@ <h2 id="the-basics">2 The Basics</h2><div class="nav"><a href="#feedback">Previo
 <p>In this section, you&#8217;ll learn a bit about <abbr title="Generic Syntax Highlighter">GeSHi</abbr>, how it works and what it uses, how to install it and how to use
 it to perform basic highlighting.</p>
 
-<h3 id="getting-geshi">2.1 Getting <abbr title="Generic Syntax Highlighter">GeSHi</abbr> work</h3><div class="nav"><a href="#the-basics">Previous</a> | <a href="#the-basics">Top</a> | <a href="#requirements">Next</a></div>
+<h3 id="getting-geshi">2.1 Getting <abbr title="Generic Syntax Highlighter">GeSHi</abbr> to work</h3><div class="nav"><a href="#the-basics">Previous</a> | <a href="#the-basics">Top</a> | <a href="#requirements">Next</a></div>
```
Comment (BenBE) [inline docs/geshi-doc.html]: Misses update of line 430 in TOC.

Though I'd prefer this (and all that whitespace noise) to be in a separate commit.

Comment (apjanke) [reply docs/geshi-doc.html]: Sorry about the whitespace noise; my editor put that in automatically. Will remove it.

Comment (apjanke) [reply docs/geshi-doc.html]: PR amended to update the TOC and remove the whitespace noise.
